### DISCUSSION TITLE
One cluster with multiple databases configured separately

### DISCUSSION THREAD

**Discussion by odsevs** [zalando/postgres-operator#2717] [Q&A]
Hi,
I'd like to have multiple databases within the same cluster, but with a distinct declarative configuration for each database. Is this possible?
It seems that all databases for a given cluster are configured in the matching "postgresql" custom resource. But if I create a custom resource per database, I will end up with multiple clusters.
I have like 80 (optional) microservices with a database and having 80 distinct PostgreSQL clusters seems like an unnecessary overhead both in terms of resource usage and administrative complexity.
What is the recommended way to configure PostgreSQL and the operator in that case?

**Comment by NaveedAhmad0984** [comment]
Did you get a solution to this ?
